assets/js/b24-form-interactions.js

```diff
@@ -13,9 +25,18 @@ var onFormLoadInteractions = {
       formElement.style.display = "none";
       bdy.style.overflow = "scroll";
     }
-    var b24Form = document.getElementById('bx_form_iframe_8');
     b24Form.scrolling = 'yes';
-    b24Form.style.height = '400vh';
+    b24Form.style['overflow-y'] = 'scroll';
+    formElement.style.height = (windowHeight) + 'px';
+    console.log(b24Form.style.height);
+    if(b24Form.style.height === '200px') {
```

Comment:
Could you a comments explaining these magic numbers 200, 300 and 600.
Also store any literal values in a constant variable that is wisely named.